hadoop-hdds/server-scm/src/test/java/org/apache/hadoop/hdds/scm/container/states/TestECContainerReplicaCount.java

```diff
@@ -88,10 +95,14 @@ public void testContainerMissingReplicaDueToPendingDelete() {
         registerNodes(Pair.of(IN_SERVICE, 1), Pair.of(IN_SERVICE, 2),
             Pair.of(IN_SERVICE, 3), Pair.of(IN_SERVICE, 4),
             Pair.of(IN_SERVICE, 5));
-    List<Integer> delete = new ArrayList<>();
-    delete.add(1);
-    ECContainerReplicaCount rcnt = new ECContainerReplicaCount(container,
-        replica, new ArrayList<>(), delete, 1);
+    ContainerReplicaPendingOps pendingOps =
```

Comment:
We don't need an instance of ContainerReplicaPendingOps for these tests.

Just:

```
List<ContainerReplicaOP> pending = new ArrayList<>();
pending.add(ContainerReplicaOp.create(...));
```

Note, I think the static `ContainerReplicaOp.create` method is only added in HDDS-6699 - ie not yet committed.

Reply:
I just used the create method and also extracted to a method to avoid some duplication here.